main.go

```diff
@@ -97,22 +96,21 @@ func (s *awsSQSSink) Sink(ctx context.Context, datumStreamCh <-chan sinksdk.Datu
 		s.logger.Errorf("failed to push message %v", err)
 	}
 
-	// log the failure response and return the ID of that.
+	// log the failure response and append that to responses object.
 	if len(response.Failed) > 0 {
 		s.logger.Error("failed to push message, err: %v", response.Failed)
```

Comment:
nit: I wonder if we can remove this 2-line check/log as well...

Reply:
I had keep the log in case of failure to show it in pod log, but if not required I'll remove it.

Reply:
It's fine, let's keep it for now.

Reply:
Ahh, I just removed and pushed the changes

Reply:
That's also fine :)